### PR TITLE
[Function] Create Login Function

### PR DESCRIPTION
**Pull request by kaz-m-93**
create login function

### CODE REVIEW

File: app/views/sessions/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<% provide(:title, "dashboard") %>
```
Comment (CharlotteDS) [inline app/views/sessions/index.html.erb:1]: This should not be the dashboard.
You can add a `dashboard.html.erb` in `views/users` folder.
Then in `PagesController`, you can add a condition inside `home` method. Like this:
```
def home
  if login_user != nil
    render "users/dashboard"
  end
end
```
You can check ab_july project for reference.
You can do this on the next task. Just remove `sessions/index.html.erb` for now.

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def login_user
```
Comment (CharlotteDS) [inline app/helpers/sessions_helper.rb:6]: You can use this helper on your header.
Example:
IF there is a logged in user, display username and dropdown on header
ELSE IF there is no logged in user, display "sign up" and "login" buttons on header

You can also add other helpers here.

**Review by CharlotteDS — CHANGES_REQUESTED** [review]

**Comment by kaz-m-93** [comment]
fix header, dashboard file and i added logout (i didnt make the task on asana)
asana:
https://app.asana.com/0/1199094899705013/1199094899705043

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -1,2 +1,8 @@
 module UsersHelper
+  def gravatar_for(user)
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+
+    image_tag(gravatar_url, alt: user.name, class: "gravatar")
+  end
```
Comment (CharlotteDS) [inline app/helpers/users_helper.rb:7]: No need to add method to get gravatar. We will use image upload for the user's picture on this project.
You can remove this method.

**Review by CharlotteDS — CHANGES_REQUESTED** [review]

**Comment by kaz-m-93** [comment]
delete gravatar method
asana:
https://app.asana.com/0/1199094899705013/1199094899705043

**Review by CharlotteDS — APPROVED** [review]

**Pull request merged**